src/librbd/image/RemoveRequest.cc

```diff
@@ -621,6 +627,24 @@ void RemoveRequest<I>::handle_remove_v1_image(int r) {
   if (!m_old_format) {
     remove_v2_image();
   }
+
+}
+
+template<typename I>
+void RemoveRequest<I>::remove_image_cache() {
```

Comment:
I am thinking I should remove image_cache when this image is removed by 'rbd rm', so add function here.